app/models/purchase.js

```diff
@@ -48,5 +48,9 @@ module.exports = (sequelize, DataTypes) => {
       )
 
       .catch(error => databaseError(error.message));
+
+  Purchase.userHasAlbum = (userId, albumId) =>
```

Comment:
This is not used anywhere else. Do not leave unused code as it can be confusing for your teamates.